gui-libs/hyprutils/hyprutils-0.1.4.ebuild

```diff
@@ -8,6 +8,10 @@ inherit cmake
 DESCRIPTION="Hyprland utilities library used across the ecosystem"
 HOMEPAGE="https://github.com/hyprwm/hyprutils"
 
+BDEPEND="
+	x11-libs/pixman
```

Comment:
BDEPEND is wrong, it's R/DEPEND, I will fix it

check https://devmanual.gentoo.org/general-concepts/dependencies/index.html

Reply:
Thank you very much